paddle/fluid/inference/tests/api/int8_mkldnn_quantization.md

```diff
@@ -0,0 +1,88 @@
+# INT8 MKLDNN quantization 
+
+This document describes how to use Paddle inference to convert the FP32 model to INT8 model on ResNet-50 and MobileNet-V1. We provide the instructions on enabling INT8 MKLDNN quantization in Paddle inference and show the ResNet-50 and MobileNet-V1 results in accuracy and performance.
+
+## 0. Prerequisite
+You need to build install at least PaddlePaddle-1.4.The test dataset and model will downloaded automatically in the build process.
+
+## 1. Enable INT8 MKLDNN quantization
+* #### Prepare the quantization warmup data 
+
+For the INT8 quantization , we should firstly  run several iterations of FP32 inference to calculate the quantization scales for each layers' inputs and outputs. FP32 running process will be executed with analysis predictor. This warmup run data is prepared for this running process.
+
+```cpp
+std::vector<std::vector<PaddleTensor>> input_slots_all;
+SetInput(&input_slots_all); //user create the SetInput function to load the data from the file
+```  
+
+* #### Create quantize config by analysis config
+
+We enable the MKLDNN quantization procedure by analysis config, and pass the warmup data we prepared before. Meanwhile, you can set all the quantization parameters by using quantize config. For example, set quantization op names, set quantization strategies etc.
+
+```cpp
+cfg.EnableMkldnnQuantizer();//Enable MKLDNN quantization
+cfg.mkldnn_quantizer_config()->SetWarmupData(warmup_data); //use analysis config to call the MKLDNN quantization config
+cfg.mkldnn_quantizer_config()->SetWarmupBatchSize(100);
+```
+
+* #### Run the accuracy compare test and profile test
```

Comment:
How about `* #### Run the accuracy compare test and profile test`->`### Run the accuracy compare test and profile test`?
![image](https://user-images.githubusercontent.com/6836917/55137142-270c9f80-516b-11e9-91c9-260f0f100566.png)
the format and indent of `Run the accuracy compare test and profile test` and `To compare the top 1 accuracy before and after MKLDNN quantization
` are exactly same.

Reply:
Remove this part